docs/translations/api-docs/aspect-ratio/aspect-ratio.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@hbjORbj I think this file should not be created.

Reply:
Done. Created from running scripts while the PR was incomplete.